public/scripts/dom-utils.js

```diff
@@ -1,49 +1,51 @@
-export const hideElement = ($el) => {
-  $el.classList.add('hidden');
-  $el.setAttribute('aria-hidden', 'true');
+export const hideElement = ($element) => {
+  $element.classList.add('hidden');
+  $element.setAttribute('aria-hidden', 'true');
 };
 
-export const showElement = ($el) => {
-  $el.classList.remove('hidden');
-  $el.removeAttribute('aria-hidden');
+export const showElement = ($element) => {
+  $element.classList.remove('hidden');
+  $element.removeAttribute('aria-hidden');
 };
 
-export const toggleVisibleElement = ($el) => {
-  $el.classList.toggle('hidden');
-  $el.toggleAttribute('aria-hidden');
+export const toggleVisibleElement = ($element) => {
+  $element.classList.toggle('hidden');
+  $element.toggleAttribute('aria-hidden');
 };
 
-export const replaceChildren = ($el, newChildren, nFirstChildren) => {
+export const replaceChildren = ($element, newChildren, nFirstChildren) => {
   nFirstChildren =
+    // eslint-disable-next-line unicorn/explicit-length-check
     Math.min(nFirstChildren, newChildren.length) || newChildren.length;
   const firstChildren = newChildren.slice(0, nFirstChildren);
-  $el.innerHTML = '';
-  firstChildren.forEach((node) => $el.appendChild(node));
+  $element.innerHTML = '';
+  for (const node of firstChildren) $element.append(node);
   setTimeout(() => {
-    newChildren.slice(nFirstChildren).forEach((node) => $el.appendChild(node));
+    for (const node of newChildren.slice(nFirstChildren)) $element.append(node);
```

Comment:
Why are you using `append` here instead of `appendChild`? The method `append` also accepts strings, which could be less safer in a typed context and prone to errors when testing using mocks.

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v51.0.1/docs/rules/prefer-dom-node-append.md